tests/conftest.py

```
@@ -279,21 +279,110 @@ def pytest_terminal_summary(terminalreporter, exitstatus, config):
+                context.tracing.stop(path=str(trace_path))
+            except Exception as exc:  # noqa: BLE001
+                warnings.warn(
+                    f"Failed to stop tracing for {safe_nodeid}: {exc}",
+                    UserWarning,
+                    stacklevel=2,
+                )
+        else:
+            try:
+                context.tracing.stop()
+            except Exception:
+                pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)